src/types/ZoomAPIConfig.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Configuration description for Zoom API
+ * @author Gabe Abrams
+ */
```

Comment:
I already made this comment in the /lib/types area, but I'm curious why the params are described when the class uses the config type and not down here in the actual type file. I'm assuming it's to do with the way the js.html docs are complied.

Reply:
You're spot on. I'm happy to explain our build/doc generation process if that helps, but it might be better to do via zoom